### ISSUE TITLE
Release Twisted 16.4.1

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8796]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8796|
|Type|task|
|Created|2016-08-28 15:13:37Z|
|Branch|https://github.com/twisted/twisted/tree/release-16.4.1-8796|

<details><summary>Searchable metadata</summary>

```
trac-id__8796 8796
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__release_16_4_1_8796 release-16.4.1-8796
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__None None
time__1472397217028321 1472397217028321
changetime__1473874559487084 1473874559487084
version__None None
owner__hawkowl hawkowl
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

Diff between 16.4.0 and 16.4.1: https://github.com/twisted/twisted/compare/twisted-16.4.0...release-16.4.1-8796

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

Builders running; you can ignore failures related to the asyncio reactor, they landed after this version was cut.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_5"></a>|@glyph set owner to @hawkowl|
|-|-|

Looks pretty straightforward.  Sorry for the delay.  Please proceed.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl set status to `closed`|
|-|-|

In changeset a7cbc2fa8b5560899e32cf9bed766d9f3cacc90a
```
#!CommitTicketReference repository="" revision="a7cbc2fa8b5560899e32cf9bed766d9f3cacc90a"
Merge release-16.4.1-8796: Release Twisted 16.4.1

Author: hawkowl
Reviewer: glyph
Fixes: #8796
```
